site/docs/guidelines/filtering.mdx

```diff
@@ -107,10 +107,13 @@ A select is useful in filtering when:
 *   Reducing data by pre-defined segment of data (E.g. In a report select 'Administrator Report' or by comment sentiment, 'Positive', 'Neutral', 'Negative')
 *   Shows the options available for that field (E.g. Country demographic, 'Australia', 'England', 'United States')
 *   Reducing data in 1 field e.g. "Department" is a "Field" and "Marketing Design" is the value
+*   Narrowing data by by a pre-defined list of options e.g. Using a searchable select element for "City" where you can narrow to a city or department by starting to type characters to narrow and select from the the available options 
+
 
 The select filter:
 
 *   Could be live-updating, could have "Apply", "Cancel", "Reset" options
+*   A select element may itself be 'searchable' and use autocomplete to narrow down from a large number of options, but the value filtered by the selected  predefined option (not the search string used to narrow those options) 
```

Comment:
double space after "selected"

Reply:
Ah, fell victim to line wrap - good catch 👍